### PR TITLE
Mothpeople can now use flight potions

### PR DESCRIPTION
**Pull request by ChesterTheCheesy**
# General Documentation
### Intent of your Pull Request
Decided to finally code this shit myself.
https://forums.yogstation.net/threads/mothpeople-flight-potion-interaction.22707/
Anyway, after drinking the flight potion, mothpeople can now extend their wings to fly exactly like people fly using angel wings.
Burned wings cannot fly, just like you can't use them to move in no gravity.
All sprites were done by me.

### Why is this change good for the game?
It's an extra feature that makes 100% sense, mothpeople have wings, why shouldn't they be able to fly?

### Briefly describe your PR and the impacts of it, in layman's terms.
Adds an extended state of all moth wings. Yes, every each moth wing has it's own sprite, so you'll still be very colorful.
The sprites i made were created based on real moth pictures.

### What should players be aware of when it comes to the changes your PR is implementing?
That they can now utilize flight potions as mothpeople.

### What general grouping does this PR fall under? 
general feature to mothpeople

# Changelog
:cl:  
rscadd: Added the ability for mothpeople to use flight potions and extend their wings 
imageadd: Added sprites for moth wings in their "open" state
/:cl:

obraz

### CODE REVIEW

**Comment by patpol4** [comment]
Those sprites are hot

**Comment by JamieD1** [comment]
@ChesterTheCheesy 

image

Fix

**Comment by ChesterTheCheesy** [comment]
I did what you asked for

**Review by Partheo — APPROVED** [review]
